accounts/abi/abi.go

```diff
@@ -137,11 +137,11 @@ func (abi *ABI) UnmarshalJSON(data []byte) error {
 
 // MethodById looks up a method by the 4-byte id
 // returns nil if none found
-func (abi *ABI) MethodById(sigdata []byte) *Method {
+func (abi *ABI) MethodById(sigdata []byte) (*Method, error) {
```

Comment:
Maybe we could use `[4]byte` here to compile time ensure the size is ok?

Reply:
We could, but then the caller needs to convert his slice of bytes into an array instead, with some for-loop. Is that really nicer?